pkg/render/render_test.go

```diff
@@ -89,7 +89,8 @@ func TestDir(t *testing.T) {
 	g := NewGomegaWithT(t)
 
 	d := MakeRenderData()
-	d.Funcs["fname"] = func(s string) string { return s }
+	d.Funcs["fname"] = func(s string) string { return "test-" + s }
```

Comment:
this looks like a significant change to me, can you explain please?

Reply:
Not sure why it was changed. I reverted it